config.default

```diff
@@ -47,6 +47,15 @@ seaIceStreamsFileName = streams.cice
 # names of ocean and sea ice meshes (e.g. EC60to30, QU240, RRS30to10, etc.)
 mpasMeshName = mesh
 
+# The system has a limit to how many files can be open at one time.  By
+# default, xarray attempts to open all files in a data set simultaneously.
+# A new option allows files to be automatically closed as a data set is being
+# read to prevent hitting this limit.  Here, you can set what fraction of the
+# system limit of open files an analysis task is allowed to use.  Note: In the
```

Comment:
is there a typo somewhere on the line above? I do not understand the sentence starting with `Here, you can set...`.